yt/geometry/coordinates/spherical_coordinates.py

```diff
@@ -277,27 +285,105 @@ def image_axis_name(self):
     def period(self):
         return self.ds.domain_width
 
+    def _get_poloidal_bounds(self):
+        ri = self.axis_id["r"]
+        ti = self.axis_id["theta"]
+        rmin = self.ds.domain_left_edge[ri]
+        rmax = self.ds.domain_right_edge[ri]
+        thetamin = self.ds.domain_left_edge[ti]
+        thetamax = self.ds.domain_right_edge[ti]
+        corners = [
+            (rmin, thetamin),
+            (rmin, thetamax),
+            (rmax, thetamin),
+            (rmax, thetamax),
+        ]
+
+        def to_poloidal_plane(r, theta):
```

Comment:
should this be defined externally?  or as a method?

Reply:
I wrote it as a method initially but it felt a little awkward since it's really only meant as a small helper function. Because its use is so narrow I figured I might as well keep it in the only place it's ever going to be used so as to keep the code tidier. What would be the benefits of keeping it separated ?